docs/faq.rst

```diff
@@ -398,6 +398,52 @@ Two mailing lists.
+
+* **Atomicity**: If any part of the transaction fails, the entire operation rolls back.
+* **Consistency**: The database remains structurally sound with every transaction.
+* **Isolation**: Each transaction is independent of other transactions.
+* **Durability**: All transaction results are permanently preserved.
+
+ACID compliance is a complex and often contested topic. The consistency guarantee is incredibly difficult to deliver in a globally distributed database topology that involves multiple clusters, each containing many nodes.
+
+For this reason, ACID-compliant databases are usually very expensive and difficult to scale. Since SQL databases are all ACID compliant to varying degrees, they also share these downsides. 
+Some relational database systems enable ACID guarantees to be relaxed in the interest of offsetting these downsides. Achieving resilient, distributed SQL database deployments can be extraordinarily difficult and expensive.
+In contrast to SQL’s ACID guarantees, NoSQL databases provide BASE guarantees:
+
+* **Basic Availability**: Data is available most of the time, even during a partial system failure.
+* **Soft state**: Replicas are not consistent all the time.
+* **Eventual consistency**: Data will become consistent at some point in time, with no guarantee when.
+
+This is also related to the `CAP theorem <https://groups.google.com/d/forum/scylladb-users>`_, which states that in a distributed data system, only two out of the following three guarantees can be satisfied:
+
+* **Consistency**: The same response is given to all identical requests
+* **Availability**: Requests receive responses even during a partial system failure
+* **Partition** Tolerance: Operations remain intact even when some nodes are unavailable.
+
+ScyllaDB, as well as `Apache Cassandra <https://cassandra.apache.org/>`_ and other NoSQL databases, sacrifice a degree of consistency in order to increase availability. Rather than providing strong consistency, they provide `eventual consistency <https://www.scylladb.com/glossary/eventual-consistency/>`_. This means that in some cases, a read request will fail to return the result of the latest WRITE. 
+In ScyllaDB (and Apache Cassandra), consistency is tunable, meaning that for a given query, the client can specify a `Consistency Level <https://university.scylladb.com/courses/scylla-essentials-overview/lessons/architecture/topic/consistency-level-cl/>`_, which refers to the number of replicas that are required to respond to a request for it to be considered successful.  This is also known as `Tunable Consistency <https://docs.scylladb.com/stable/glossary.html#term-Tunable-Consistency>`_. 
```

Comment:
This paragraph is missing the most important part - the answer to your FAQ question, which was about eventual consistency. I think you should explicitly say that some choices (e.g, read and write to a quorum of nodes) allow to achieve strong consistency, and others provide eventual consistency. Unless you disagree that this is called "strong consistency". (if you disagree, why do you disagree?).
We do have a bug in this "strong consistency" though - regarding timestamp ties - that might be worth discussing.
BTW, my comment here is only about consistency - the C part of ACID, not the I. We have other isolation problems (of concurrent requests) and LWT (below) solves those.

Reply:
Good point, I added an explanation here, specifically pertaining to the question. 
I avoided discussing if strong consistency can be achieved with ScyllaDB (or with Cassandra), as I didn't want to get into this as it is complex and probably out of scope for the average user. It would require another definition of (monotonic read) consistency, dealing with timestamps, multiple datacenters, and more. 
Feel free to add this part if you think it's relevant.